test/e2e/dr/common.go

```diff
@@ -0,0 +1,473 @@
+	return string(out)
+}
+
+func scpFileToHost(src string, proxy string, dest string, destHost string) {
+	e2e.Logf("Copying %s to %s at host '%s' via %s", src, dest, destHost, proxy)
+
+	command := fmt.Sprintf(scpToHostTemplate, sshOpts, proxy, src, destHost, dest)
+	runCommandAndRetry(command)
+}
+
+func scpFileFromHost(src string, srcHost string, proxy string, dest string) {
+	e2e.Logf("Copying %s from '%s' to %s via %s", src, srcHost, dest, proxy)
+
+	command := fmt.Sprintf(scpFromHostTemplate, sshOpts, proxy, srcHost, src, dest)
+	runCommandAndRetry(command)
+}
+
+func runViaBastionSSH(host string, proxy string, remoteCommand string) string {
+	e2e.Logf("Running '%s' on host %s via %s", remoteCommand, host, proxy)
+
+	command := fmt.Sprintf(sshTemplate, sshOpts, proxy, host, remoteCommand)
+	return runCommandAndRetry(command)
+}
+
+func getAllMasters(oc *exutil.CLI) []string {
+	nodeNames := sets.NewString()
+
+	e2e.Logf("Fetching a list of masters")
+
+	masterNodes, err := oc.AdminKubeClient().CoreV1().Nodes().List(metav1.ListOptions{
+		LabelSelector: "node-role.kubernetes.io/master",
+	})
+	for i := range masterNodes.Items {
+		node := &masterNodes.Items[i]
+		nodeNames.Insert(node.ObjectMeta.Name)
+	}
+
+	o.Expect(err).NotTo(o.HaveOccurred())
+
+	return nodeNames.List()
+}
+
+func constructEtcdConnectionString(masters []string, proxy string) string {
+	//TODO vrutkovs: replace this nonsense with `etcdctl member list -w json ...`
+	etcdConnectionString := ""
+	e2e.Logf("Construct etcd connection string")
+	for _, master := range masters {
+		etcdEnv := runViaBastionSSH(master, proxy, "cat /run/etcd/environment")
+		var entry string
+		for _, entry = range strings.Split(etcdEnv, "\n") {
+			if strings.HasPrefix(entry, "ETCD_DNS_NAME=") {
+				break
+			}
+		}
+		etcdDNSName := strings.Split(entry, "=")[1]
+		o.Expect(etcdDNSName).NotTo(o.BeEmpty())
+		etcdConnectionString = fmt.Sprintf("%setcd-member-%s=https://%s:2380,", etcdConnectionString, master, etcdDNSName)
+	}
+	return etcdConnectionString[:len(etcdConnectionString)-1]
+}
+
+func removeSSHBastion(oc *exutil.CLI) {
+	e2e.Logf("Removing ssh bastion")
+	err := oc.AdminKubeClient().CoreV1().Namespaces().Delete(bastionNamespace, &metav1.DeleteOptions{})
+	o.Expect(err).NotTo(o.HaveOccurred())
+}
+
+func waitForMastersToUpdate(oc *exutil.CLI, mcps dynamic.NamespaceableResourceInterface) {
+	e2e.Logf("Waiting for MachineConfig master to finish rolling out")
+	err := wait.Poll(30*time.Second, 30*time.Minute, func() (done bool, err error) {
+		return upgrade.IsPoolUpdated(mcps, "master")
+	})
+	o.Expect(err).NotTo(o.HaveOccurred())
+}
+
+func waitForOperatorsToSettle(coc dynamic.NamespaceableResourceInterface) {
+	var lastErr error
+	// gate on all clusteroperators being ready
+	available := make(map[string]struct{})
+	lastErr = nil
+	var lastCOs []objx.Map
+	wait.PollImmediate(30*time.Second, operatorWait, func() (bool, error) {
+		obj, err := coc.List(metav1.ListOptions{})
```

Comment:
you should have access to `client-go/config/v1`

Reply:
That was copied from upgrade test, but it appears it was updated since then. I'll updated DR part later on, thanks!